### PR TITLE
L7: project a robot

### PR DESCRIPTION
**Pull request by Brandonz07**

### CODE REVIEW

File: src/br/js/L7-project-a-robot/Persistent_group.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    add(value) {
+        if (!this.has(value)) {
+        return new PGroup(this.members.concat([value]));
+        }
+        return this;
+    }
+
+    delete(value) {
+        if (this.has(value)) {
+        return new PGroup(this.members.filter(v => v !== value));
```
Comment (carin91) [inline src/br/js/L7-project-a-robot/Persistent_group.js:15]: use a more descriptive name for v

Comment (carin91) [inline src/br/js/L7-project-a-robot/Persistent_group.js:15]: create a new static method from and refactor this line with it.

**Review by carin91 — CHANGES_REQUESTED** [review]

File: src/br/js/L7-project-a-robot/Persistent_group.js (hunk 1 above)
Comment (Brandonz07) [reply src/br/js/L7-project-a-robot/Persistent_group.js]: i changed name

Comment (Brandonz07) [reply src/br/js/L7-project-a-robot/Persistent_group.js]: static method

**Review by carin91 — APPROVED** [review]

**Pull request merged**